.github/workflows/e2e.maven.workflow_dispatch.main.default.slsa3.yml

```
@@ -0,0 +1,49 @@
+name: Maven e2e test - simple
+on:
+  schedule:
+    - cron: "0 6 * * *"
+  workflow_dispatch:
+
+permissions: read-all
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  bootstrap:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - run: mv e2e/maven/pom.xml ./ && cp -r e2e/maven/src ./ && rm -r e2e/maven/src
+  build:
+    #runs-on: ubuntu-latest
+    permissions:
+      id-token: write # For signing.
+      contents: read # For repo checkout of private repos.
+      actions: read # For getting workflow run on private repos.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/builder_maven_slsa3.yml@main
+  verify:
```

nit: add extra space

```suggestion
    uses: slsa-framework/slsa-github-generator/.github/workflows/builder_maven_slsa3.yml@main

  verify:
```